src/components/VenderDashboard/VenderDashboard.tsx

```diff
@@ -0,0 +1,40 @@
+import './VenderDashboard.css';
+import WeddingCard from '../WeddingCard/WeddingCard'
+import {individualWedding, weddings} from '../../weddingData'
+
+
+function VenderDashboard() {
+  const weddingCards = weddings.map((wedding, index) => {
+    return (
+      <WeddingCard
+        key={wedding.weddingId}
+        name={wedding.name}
+        image={wedding.image}
+        date={wedding.date}
+      />
+    )
+  })
+    return (
+        <section className="dashboardWrapper">
+          <section className="optionsWrap">
+          <div className="button" id="addWeddingButton">
+            <div id="translate"></div>
+            <a>Add A Wedding</a>
+          </div>
+            <section className="filterWrap">
+            <label className="label">Filter By :</label>
+            <select className="button">
+                <option value="0">All</option>
+                <option value="1">Upcoming</option>
+                <option value="2">Past</option>
+                <option value="3">Today</option>
+            </select>
+            </section>
+          </section>
+          <section className="weddingCardWrap">
+            {weddingCards}
+          </section>
+        </section>
+    )
+}
+export default VenderDashboard;
```

Comment:
The weddingCard component looks good! I'm sure later on we will need to identify types if we are doing anything with these cards.

Reply:
> We have a dashboard! I'll add a couple of tickets with specific next steps (functionality for various buttons, and small spelling error - thing `vender` should be `vendor`

I think both of these spellings are technically correct. We should definitely make sure to establish which one we want to use and make sure it's a norm across the board. Good call, Hanna!